pilot/pkg/bootstrap/server.go

```diff
@@ -902,7 +903,8 @@ func (s *Server) getCertKeyPaths(tlsOptions TLSOptions) (string, string) {
 	return key, cert
 }
 
-// getRootCertificate returns the root certificate from TLSOptions if available or from ca.
+// getRootCertificate returns the root certificate from TLSOptions if available or from ca, or from SPIFFE bundle
```

Comment:
Plural - or 'the set of root certificates'

We have a pervasive problem of ignoring cert rotation, migration and multiple roots - at least the docs should 
be clear.

Reply:
And the method name could have an 's' at the end - or getRootCertPool - the return value is a set of certificates, indexed by subject and subject key id.